src/PostgrestBuilder.ts

```diff
@@ -149,7 +149,7 @@ export default abstract class PostgrestBuilder<Result>
           }
         }
 
-        if (error && this.isMaybeSingle && error?.details?.includes('Results contain 0 rows')) {
+        if (error && this.isMaybeSingle && error?.details?.includes('0 rows')) {
```

Comment:
Is there a reason why we can't rely on error codes instead? Is this the fact that `ApiRequestErrorCode16` is an umbrella error for "incorrect number of rows", and not just 0?

Reply:
Yeah, we need to distinguish 0 rows from >1 rows

Reply:
Still not a fan of it though - ideally we can check this in a more robust manner